README.md

```diff
@@ -14,8 +14,8 @@ out which program to use for what file type.
 ![screenshot](https://raw.githubusercontent.com/ranger/ranger-assets/master/screenshots/screenshot.png)
 
 This file describes ranger and how to get it to run.  For instructions on the
-usage, please read the man page.  See `HACKING.md` for development specific
-information.
+usage, please read the man-page (`man ranger` in a terminal).  See `HACKING.md`
```

Comment:
You're completely right about 'man page', doesn't make any sense to have a hyphen there. I've also looked into l72, and couldn't find anything concrete about quotes used as an element in a compound. I'm not sure if l72 constitutes a compound, though, so since we're both in doubt, I suggest dropping it.